src/string/lipogram.rs

```diff
@@ -0,0 +1,93 @@
+use std::collections::HashSet;
+
+//*
+// Fn that returns the letters that are missing from the input slice
+// and are present in the english alphabet
+fn compute_missing(in_str: &str) -> String {
+    let alphabet: HashSet<char> = "abcdefghijklmnopqrstuvwxyz".chars().collect();
+
+    let letters_used: HashSet<char> = in_str
+        .to_lowercase()
+        .chars()
+        .filter(|c| c.is_ascii_alphabetic())
+        .collect();
+
+    let difference_set: HashSet<char> = alphabet.difference(&letters_used).cloned().collect();
+
+    difference_set
+        .into_iter()
+        .map(|c| c.to_string())
+        .collect::<Vec<String>>()
+        .join("")
+}
+
+//*
+// Fn that checks if the slice is a lipogram.
+// Lipogram - sentence in which a particular letter or group of letters is avoided
+pub fn is_lipogram(lipogram_str: &str) -> bool {
+    let unused_letters = compute_missing(lipogram_str);
+
+    !unused_letters.is_empty()
+}
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn test_lipogram_invalid1() {
```

Comment:
Is there a better name of these tests? Maybe
```suggestion
    fn test_not_lipogram_1() {
```